pysph/tools/ipy_viewer.py

```diff
@@ -237,7 +251,9 @@ class Viewer2DWidgets(object):
 
     def __init__(self, file, file_count):
 
-        self.temp_data = load(file)['arrays']
+        self.temp_data = load(file)
```

Comment:
I don't like the use of "file" as it overwrites the builtin `file` type maybe use `filename`?

Reply:
Changed to `file_name`.